build/azure-pipelines.yml

```diff
@@ -24,14 +24,14 @@ stages:
       matrix:
         Packages:
           ApplicationConfiguration: Release
-          ApplicationPlatform: NuGet
```

Comment:
So it's no longer aligned with the other recipes?

Reply:
The other recipes had this because of Tests/Sample projects inside the solution. This one is doesn't need it. Also I think it could be done better our recipe is quit out dated. I was reading the Microsoft documentation and we could have something smaller than that.

See https://learn.microsoft.com/en-us/azure/devops/pipelines/artifacts/nuget?view=azure-devops&tabs=yaml for more details.